jenkins/agent-ctl.sh

```diff
@@ -39,12 +39,12 @@ usage() {
   echo '  update-image: update the image-family used to create new disks'
   echo 'Common commands:'
   echo '  # Refresh image'
-  echo '  $(basename "${0}") --base-image light-agent auto-image'
+  echo "  $(basename "${0}") --base-image light-agent auto-image"
   echo '  # Retire agent'
```

Comment:
Style guide habit? We are supposed to always favor single quotes except where variable substitution is needed